ch.vorburger.osgi.gradle/src/main/java/ch/vorburger/osgi/gradle/internal/SourceInstallServiceImpl.java

```diff
@@ -67,7 +69,14 @@ public SourceInstallServiceImpl(BundleContext bundleContext) {
                 installFuture.setException(e);
             }
         };
-        ListenableFuture<Void> buildFuture = buildService.buildContinously(projectDirectory, "build", listener);
+
+        ListenableFuture<Void> buildFuture;
+        if (new File(projectDirectory, "pom.xml").exists()) {
+            buildFuture = mavenBuildService.buildContinously(projectDirectory, "install", listener);
```

Comment:
this is really cool... but now I should rename this project from ch.vorburger.osgi.gradle to ... ch.vorburger.osgi.builder? :smiley_cat: (I won't.)

Reply:
yeah the name of the project is no longer accurate anymore